src/installer/mod.rs

```diff
@@ -6,7 +6,7 @@ pub mod destination;
 pub mod error;
 pub mod executable;
 mod executable_file_installer;
-mod file;
+pub mod file;
```

Comment:
Revert all changes to this file since they are not needed. See comments in `download_handler.rs`